release_notes/ocp-4-8-release-notes.adoc

```diff
@@ -193,6 +193,24 @@ In the table, features are marked with the following statuses:
 [id="ocp-4-8-bug-fixes"]
 == Bug fixes
 
+*Bare Metal Hardware Provisioning*
+
+* Previously, Ironic failed to download an image for installation because it uses HTTPS by default and Ironic does not have the correct certificate bundle available. This issue is fixed by setting the image download as insecure to request a transfer without the certificate. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1953795[*BZ#1953795*])
+
+* Previously, when using dual stack deployments, worker node hostnames sometimes did not match the name inspected before deployment. This caused nodes to need manual approval. This has been fixed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1955114[*BZ#1955114*])
+
+* Previously, in UEFI mode, the ironic-python-agent created a UEFI bootloader entry after downloading the RHCOS image. When using an RHCOS image based on RHEL 8.4, the image could fail to boot using this entry. If the entry installed by Ironic was used when booting the image, the boot could fail and output a BIOS error screen. This is fixed by the ironic-python-agent configuring the boot entry based on a CSV file located in the image, instead of using a fixed boot entry. The image boots properly without error. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1966129[*BZ#1966129*])
+
+* Previously, a node sometimes selected the incorrect IP version upon startup (v6 instead of v4, or vice versa). The node would fail to come up because it did not receive an IP address. This is fixed by cluster-baremetal-operator passing the IP Option to the downloader (ip=dhcp or ip=dhcp6), so this is set correctly at startup and the node comes up as expected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1946079[*BZ#1946079*])
```

Comment:
Please use IPv4 and IPv6--it's not mysterious without, it's just consistent.

Please use "fail to start because" instead of "fail to come up because".  Formality isn't bad and it *might* translate better.

Operator names are treated as [proper nouns](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#operator-name-capitalization) according to the guidelines, so I expected "the Cluster Bare Metal Operator" instead of "cluster-baremetal-operator".

Please lowercase "option" so we get "IP option" and also enclose each of the sample arguments in backticks to format them as [inline code](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#inline-code-or-commands) like "...(`ip=dhcp` or `ip=dhcp6`),..."